src/background.js

```diff
@@ -1159,7 +1159,12 @@ adblocker.on('enabled', () => {
 				before: ['checkBlocklist']
 			}),
 			adblocker.action('addWhiteListCheck',
-				url => isWhitelisted({ sourceUrl: url }))
+				url => isWhitelisted({
+					sourceUrlParts: {
```

Comment:
@sammacbeth can we write this as `{ sourceUrlParts: new URL(url) }` instead? A bit cleaner, so we don't do `sourceUrlParts.host.host` later on?

Reply:
@jsignanini You are right, it's not really nice, but unfortunately this is needed (at the moment) to mimic some internal data structure that is expected by this step (there is indeed a `host.host` at some place in the code). We will probably try to clean this up to make the structure of this object more consistent.